<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>66</green>
        <blue>63</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>66</green>
        <blue>63</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>66</green>
        <blue>63</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50,50,50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>104</red>
         <green>106</green>
         <blue>97</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>76</red>
         <green>76</green>
         <blue>76</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>104</red>
         <green>106</green>
         <blue>97</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>76</red>
         <green>76</green>
         <blue>76</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>104</red>
         <green>106</green>
         <blue>97</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>76</red>
         <green>76</green>
         <blue>76</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>50</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>370</y>
      <width>271</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QListView" name="LISTVIEW_NETWORKTLOG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="QPIXMAX_IMAGE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1201</width>
      <height>941</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="GROUPBOX_IMAGESIZE">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>220</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Size</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>15</number>
       </property>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="COMBOBOX_IMAGESIZE">
       <property name="styleSheet">
        <string notr="true">background rgb(255, 255, 255)</string>
       </property>
       <item>
        <property name="text">
         <string>x 0.2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x 0.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x 0.7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x 0.9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x 1.0</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GROUPBOX_FACEDETECT">
       <property name="title">
        <string>FaceDetect</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="FACEDETECTLABEL_FACETHRESHOLD">
          <property name="text">
           <string>Face Threshold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="FACEDETECTHSLIDER_FACETHREAHOLD">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>1221</x>
      <y>10</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="TOOLBAR_FILE">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="FILE_SAVEIMAGE"/>
    <addaction name="FILE_LOADIMAGE"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>Network</string>
    </property>
    <addaction name="NETWORK_CONNECT"/>
    <addaction name="NETWORK_DISCONNECT"/>
   </widget>
   <widget class="QMenu" name="menuCam">
    <property name="title">
     <string>Cam</string>
    </property>
    <addaction name="CAM_PLAY"/>
    <addaction name="CAM_STOP"/>
   </widget>
   <widget class="QMenu" name="menuLearn">
    <property name="title">
     <string>Learn</string>
    </property>
    <addaction name="LEARN_IMAGE"/>
    <addaction name="LEARN_MAKECSV"/>
    <addaction name="LEARN_GetFaceImages"/>
   </widget>
   <addaction name="TOOLBAR_FILE"/>
   <addaction name="menuNetwork"/>
   <addaction name="menuCam"/>
   <addaction name="menuLearn"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="FILE_LOADEIMAGE">
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="FILE_SAVEIMAGE">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="NETWORK_CONNECT">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="NETWORK_DISCONNECT">
   <property name="text">
    <string>DisConnect</string>
   </property>
  </action>
  <action name="CAM_CONNECT">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="CAM_PLAY">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="CAM_STOP">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="CAMDISCONNECT">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="FILE_LOADIMAGE">
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="LEARN_IMAGE">
   <property name="text">
    <string>Learn Image</string>
   </property>
  </action>
  <action name="FILE_MAKECSV">
   <property name="text">
    <string>Make csv file(dumy images)</string>
   </property>
  </action>
  <action name="LEARN_MAKECSV">
   <property name="text">
    <string>Make csv file(Images)</string>
   </property>
  </action>
  <action name="LEARN_GetFaceImages">
   <property name="text">
    <string>Get Face(Images)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
